doc/secure-boot/secure-boot-test-plan.md

```diff
@@ -0,0 +1,115 @@
+# SONiC Secure Boot Test Plan
+
+## Overview
+The purpose of this test plan is to validate that SONiC switch supports secure boot.
+
+## Test cases
+
+### 1. Test to install an signed image
+Test to install a signed image, expect the signed image can be installed.
+
+**Steps:**
+ * Copy a signed image
+ * Run the sonic installer command: sonic_installer install <image>
+ * Expect the installation successfully
+
+### 2. Test to install an unsigned image
+Test to install an unsigned image, expect the unsigned image never be installed when SONiC Secure Boot enabled.
+
+**Steps:**
+ * Create a fake unsigned image
+ * Run the sonic installer command: sonic_installer install <image>
+ * Expect the installation failed with image not signed
+
+### 3. Test fast-reboot/warm-reboot from signed image into another signed image
+
+**Steps:**
+ * Basic boot up test
+ * Allowlist not cover this scenario.
+ * Check /proc/cmdline
```

Comment:
`Check for secure_boot_enable=y in /proc/cmdline`
This might be a bit more precise as to what needs to be checked for.

Reply:
Fixed